backend/backend.py

```diff
@@ -32,6 +33,20 @@ def __init__(self, width: int, height: int, publish_recording_status_func: Publi
         # Time between recording statuses, in seconds
         self.recording_status_interval = 60
 
+        # time that we last called self.send_camera_is_online
+        self.prev_camera_is_online_time = 0
+        # Time between recording statuses, in seconds
+        self.camera_is_online_interval = 60
```

Comment:
That would not accurately reflect what the variable does since it represents the last time the function was called, not the last time the camera was online.